package.json

```diff
@@ -18,7 +18,8 @@
     "test:integration": "lerna exec --scope integration-tests -- yarn sauce",
     "test:performance": "lerna exec --scope perf-benchmarks -- best --runner default",
     "release:ci:npm": "./scripts/release_npm.sh",
-    "release:ci:dist-tags": "./scripts/release_dist-tags.sh"
+    "release:ci:dist-tags": "./scripts/release_dist-tags.sh",
+    "release:version": "lerna version --exact --force-publish --no-push"
```

Comment:
>> We also might want to add allowBranch config to prevent releasing on a branch different than master or a release branch.

My thinking was that we want the flexibility to produce alpha releases from any branch but now that I think about it more, maybe we shouldn't be pushing tags that aren't pointing to a protected branch. Although we do this today, having tags that point to commits that are not guaranteed to exist at a later time due to squashed merges or deleted branches seems like a bad idea. Thoughts?

>> Why passing --no-push don't we want to push all the newly created a commit and the associated tag when doing a release?

This seems fine if we restrict pushing versions to protected branches.

>> With this change, we will be producing the release commit from our machine. We should prevent at this point having uncommited files before doing the release to avoid broken releases (via preversion hook probably).

I don't think this is a concern since lerna will update the version numbers in all the package.json files, commit those files, and tag that commit.